documentation/modules/advanced/pages/jwt.adoc

```diff
@@ -0,0 +1,168 @@
+= End-user authentication with JWT
+include::_attributes.adoc[]
+
+[IMPORTANT]
+.Before Start
+====
+You should have NO virtualservice, destinationrule, gateway or policy (in `tutorial` namespace) `istioctl get virtualservice` `istioctl get destinationrule` `istioctl get gateway` `istioctl get policy`
```

Comment:
it will be nice if we say why we want user to clean things up e.g to show only what is needed in this context

Reply:
Done